_episodes/10-shielding.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Live Coding Shielding Problem"
+teaching: 0
+exercises: 90
+questions:
+- "How can I perform a shielding calculation using Python?"
+keypoints:
+- "Use Python to write a simple shielding calculation script."
+- "Learn some best practices when writing Python."
+objectives:
+- "Produce a Python script from scratch"
+- "Execute Python scripts in a command line"
+- "Selection of appropriate data structures for an application"
+- "Understand the value of using functions"
```

Comment:
"Value using functions"

"Value" is an actual type of affective learning outcome.
Understand is vague as it does not tell us the level of understanding we want them to have, which is "valuing".